README.md

```diff
@@ -67,6 +68,19 @@ helm install ratify \
     --set-file notaryCert=./notary.crt
 ```
 
+Option 2: Install ratify with charts from your local branch.  
+Note: Latest chart in main may not be compatible with the last released version of ratify image. Weekly dev build coming soon. Please check back on issue [643](https://github.com/deislabs/ratify/issues/643)
```

Comment:
this may get updated depending on the order of merging of the PRs

Reply:
To avoid this note I was thinking it might make more sense to point to the dev build?

Reply:
sounds good, just merged the dev build PR.

Reply:
is there a label we can use for the dev build?  What should the image value to point to latest dev build?

Reply:
something like this? https://docs.github.com/en/repositories/releasing-projects-on-github/linking-to-releases

Reply:
We could link to the `ratify-dev` package and the latest one will show as the default copy command